viewing_party/party.py

```diff
@@ -1,23 +1,139 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title == None or genre == None or rating == None:
+        return None

```

Comment:
I'd suggest moving the `new_movie` declaration under the invalid data check so that we don't allocate space for `new_movie` unless we know it will be used.

```suggestion
    if title == None or genre == None or rating == None:
        return None
        
    new_movie = {}
```